tests/cli/demultiplex/test_validate_sample_sheet.py

```diff
@@ -95,13 +95,13 @@ def test_validate_correct_bcl2fastq_sample_sheet(
 def test_validate_correct_dragen_sample_sheet(
     cli_runner: CliRunner,
     sample_sheet_context: dict,
-    dragen_flow_cell_full_name: str,
-    novaseq_dragen_sample_sheet_path: Path,
+    bcl_convert_flow_cell_full_name: str,
+    novaseq_bcl_convert_sample_sheet_path: Path,
 ):
-    """Test validate sample sheet when using a Dragen sample sheet."""
+    """Test validate sample sheet when using a bcl_convert sample sheet."""
```

Comment:
```suggestion
    """Test validate sample sheet when using a BCLconvert sample sheet."""
```
OR what does bcl_convert stand for here?